internal/support/logging/init.go

```diff
@@ -73,7 +74,8 @@ func (s ServiceInit) BootstrapHandler(
 	// initialize database.
 	for startupTimer.HasNotElapsed() {
 		var err error
-		dbClient, err = getPersistence()
+		credentials, err := container.CredentialsFrom(dic.Get).GetDatabaseCredentials(Configuration.Databases["Primary"])
+		dbClient, err = getPersistence(credentials)
```

Comment:
You're not dealing with err result from the first line before overwriting it in the second.